api/management/swagger-api_management.yaml

```diff
@@ -787,6 +787,42 @@ paths:
       responses:
         200:
           description: successful operation
+  /realms/{realm}/users/{userID}/credentials/{credentialID}/unlock:
+    put:
+      tags:
+      - Credentials
+      summary: Reset a locked credential for the user (ex: paper card with too many wrong errors)
+      parameters:
+      - name: realm
+        in: path
+        description: realm name (not id!)
+        required: true
+        schema:
+          type: string
+      - name: userID
+        in: path
+        description: User id
+        required: true
+        schema:
+          type: string
+      - name: credentialID
+        in: path
+        description: Credential id
+        required: true
+        schema:
+          type: string
+      - name: type
```

Comment:
I don't see the benefit of this parameter, I would suggest to remove it to keep it simple